tests/Finals.tex

```diff
@@ -1,85 +1,77 @@
 \chapter{Finals}
-
-The competition ends with the Finals on the last day, where the two teams with the highest total score compete.
-The \iterm{Finals} are conducted as a final themed demonstration.
-
-%To avoid logistical issues during the last day of the competition, the \iterm{Finals} are divided into two sets of demonstrations: the Bronze Competition and the RoboCup @Home Grand Finale.
-%The Bronze Competition is a set of demonstrations that are carried out before the RoboCup @home Grand Finale. Here, all the leagues run in parallel, with the fourth and third highest scored teams competing for the bronze.
-%Finally, the two teams with the highest score in each League present their demonstrations in a serialized manner during the RoboCup @Home Grand Finale.
-
-Even though each league has its own first, second and third place, the \iterm{Finals} are meant to show the best of all leagues to the jury members as well as the audience and, thus, warrants a single schedule slot.
+\label{chap:finals}
+\raggedbottom
+The competition ends with the \FINAL{} on the last day, where the two teams with the highest total score compete. The \FINAL{} are conducted as an open, themed demonstration. The \FINAL{} of each league is scheduled at different times.
 
 \section{Structure and Theme}
```

Comment:
Since this section is very compact now, would recommend using \section*{} to cut down TOC entries.

Reply:
Imo it's fine like it is